eth2near/eth_rpc_client/src/beacon_rpc_client.rs

```diff
@@ -34,7 +34,7 @@ impl BeaconRPCClient {
     const URL_BODY_PATH: &'static str = "eth/v2/beacon/blocks";
     const URL_GET_LIGHT_CLIENT_UPDATE_API: &'static str = "eth/v1/beacon/light_client/updates";
     const URL_FINALITY_LIGHT_CLIENT_UPDATE_PATH: &'static str =
-        "eth/v1/beacon/light_client/finality_update/";
+        "eth/v1/beacon/light_client/finality_update";
```

Comment:
It can't be change in that way:
`https://lodestar-goerli.chainsafe.io/eth/v1/beacon/light_client/finality_update/`
`https://lodestar-goerli.chainsafe.io/eth/v1/beacon/light_client/finality_update`

For other URLs extra parameters are added and we need not `/`.

Reply:
Ok, looks like the route was changed in the last version of lodestar, In this case I will reimplement it by adding versioning for those routes

Reply:
Fixed in 02851a4c8ebe10d0f313423f44440b87a5f09e47